Library/Formula/rethinkdb.rb

```diff
@@ -75,6 +73,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    shell_output("#{bin}/rethinkdb create -d test")
```

Comment:
That error message must come from rethinkdb, seems more like a problem on that side than our logging code (which simply redirects stderr to a file).